README.md

```diff
@@ -63,6 +63,7 @@ A Collection of programming [CheatSheets](./CheatSheets/) for developers to quic
 30. [nodejs-cheatsheet.md](./CheatSheets/nodeJs-cheatsheet.md)
 31. [mongodb-cheatsheet.md](./CheatSheets/mongodb-cheatsheet.md)
 32. [extension-cheatsheet.md](./CheatSheets/extension-cheatsheet.md)
+33. [xml-cheatsheet.md](./CheatSheets/XML-cheatsheet.md)
```

Comment:
You added `XML-cheatsheet.md` file in `README.md`. But where is your `XML-cheatsheet` file? You didn't create the `XML-cheatsheet` file yet.

Please, rename your file `XML-cheatsheet.md` to `xml-cheatsheet.md`.